lib/support/requests/anonymous/anonymous_contact.rb

```diff
@@ -33,6 +30,11 @@ def requester
             only_actionable.
             order("created_at desc")
         }
+
+        private
+        def anonymous_contact_params
```

Comment:
I believe that we can drop this method (as well as `ProblemReport#problem_report_params` and the other params methods) because this app only reads these models (there aren't any controllers that write these). Some time (soon, hopefully) we can update this app to read anonymous feedback through the `support-api`, not via the DB.